### ISSUE TITLE
How to use model?

### ISSUE THREAD

**Issue by NKNguyenLX** [MaybeShewill-CV/lanenet-lane-detection#76]
Sorry, I am new to deep learning. In your test model, can you show me how to use trained lanenet model? I have used .hdf5 config but now i see there are 5 files in your trained model. I have tried all of that but nothing seems to work. Thanks!!!

**Comment by MaybeShewill-CV** [comment]
@NKNguyenLX Please follow the readme instruction to test the model

**Comment by NKNguyenLX** [comment]
I have followed the readme instruction but i dont know what to replace in this part
 --weights_path path/to/your/model_weights_file. I stored all trained model in /home/nknight/Works/git/lanenet-lane-detection/trained_model. I have tried to replace path for each file in this folder but not thing work.

**Comment by MaybeShewill-CV** [comment]
@NKNguyenLX The weights_path is the pretrained model file you download before you start to test the  lancet model. The pretrained model weights link can be found in readme file:)

**Comment by NKNguyenLX** [comment]
@MaybeShewill-CV 
I try this command:  
python3 tools/test_lanenet.py --is_batch False --batch_size 1 --weights_path trained_model/tusimple_lanenet_vgg_2018-10-19-13-33-56.ckpt-200000.data-00000-of-00001 --image_path data/tusimple_test_image/0.jpg
And the error was:
I1026 14:42:24.020394 13783 tf_logging.py:116] Restoring parameters from trained_model/tusimple_lanenet_vgg_2018-10-19-13-33-56.ckpt-200000.data-00000-of-00001
2018-10-26 14:42:24.043068: W tensorflow/core/util/tensor_slice_reader.cc:95] Could not open trained_model/tusimple_lanenet_vgg_2018-10-19-13-33-56.ckpt-200000.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

**Comment by MaybeShewill-CV** [comment]
@NKNguyenLX --weights_path trained_model/tusimple_lanenet_vgg_2018-10-19-13-33-56.ckpt-200000.data-00000-of-00001 should be --weights_path trained_model/tusimple_lanenet_vgg_2018-10-19-13-33-56.ckpt-200000. Maybe you need to learn some tensorflow knowledge first:)

**Comment by NKNguyenLX** [comment]
@MaybeShewill-CV thank you very much. Your works are great i hope one day i can train a network myself. Now is time for tensorflow basic ^^.

**Comment by MaybeShewill-CV** [comment]
@MaybeShewill-CV Thanks. Good luck:)
